docs/application/dotnet/guides/machine-learning/singleshot.md

```diff
@@ -0,0 +1,120 @@
+
+- Invoking the neural network model with input data
+
+  After setting up the SingleShot instance with its required information, you can [invoke the model with the input data and get the inference output result](#invoke).
+
+- Fetching the inference result after invoking
+
+  You can [fetch the inference result after invoking](#fetch).
+
+## Prerequisites
+
+To use Machine Learning Inference with your application:
+
+1. To use the methods and properties of the `Tizen.MachineLearning.Inference.SingleShot` class or its related classes such as `Tizen.MachineLearning.Inference.TensorsData` and `Tizen.MachineLearning.Inference.TensorsInfo`, include the [Tizen.MachineLearning.Inference]() namespace in your application:
+    ```C#
+    using Tizen.MachineLearning.Inference;
+    ```
+
+2. If the model file you want to use is located at `Media Storage` or `External Storage`,
+the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+    ```xml
+    <privileges>
+      <!--To access media storage-->
+      <privilege>http://tizen.org/privilege/mediastorage</privilege>
+
+      <!--To access, read, and write to the external storage-->
+      <privilege>http://tizen.org/privilege/externalstorage</privilege>
+    </privileges>
+    ```
+
+<a name="manage"></a>
+## Managing tensor information
+
+In this example, we use the MobileNet v1 model with TensorFlow Lite, which does image classification. Its input data type is specify bit width and its input dimension is `3 X 224 X 224`. Its output data type is same but its output dimension is `1001 X 1 X 1 X 1`.
+
+In order to configure the Tensor information, you need to create a new instance of the `Tizen.MachineLearning.Inference.TensorsInfo` class first. Then you can add each Tensor information such as its datatype, dimension and name (optional) one by one as below.
+
+```C#
+/* Input Dimension: 3 * 224 * 224 */
+TensorsInfo in_info = new TensorsInfo();
+in_info.AddTensorInfo(TensorType.UInt8, new int[4] { 3, 224, 224, 1 });
+
+/* Output Dimension: 1001 for classification */
+TensorsData out_data = new TensorsInfo();
+out_info.AddTensorInfo(TensorType.UInt8, new int[4] { 1001, 1, 1, 1 });
+```
+
+<a name="load"></a>
+## Loading the neural network model and configures a runtime environment
+
+1. Since the model file is located at the resource directory for your own application, you need to get its absolute path as below.
+
+```C#
+string ResourcePath = Tizen.Applications.Application.Current.DirectoryInfo.Resource;
+string model_path = ResourcePath + "models/mobilenet_v1_1.0_224_quant.tflite";
+```
+
+2. Then you can load the neural network model from storage and configure runtime environment with `Tizen.MachineLearning.Inference.SingleShot` class. The first parameter is the absolute path to the neural network model file and remaining 2 are input/output TensorsInfo instance. `ArgumentException` maybe raise because of invalid parameter.
+
+```C#
```

Comment:
The code block alignment is not correct. Please fix the indentation according to the numbered list.

Reply:
ACK!